assets/css/hide-amazon-express-admin.scss

```diff
@@ -0,0 +1,4 @@
+table.wc_gateways tr[data-gateway_id="amazon_payments_advanced_express"],
+.order_data_column .edit_address #_payment_method option[value="amazon_payments_advanced_express"] {
+	display: none !important;
```

Comment:
why this way of hiding and not avoid to render it?

Reply:
yea that was preferred, but couldn't find a filter unsetting this Payment Gateway in the backend...